source/renderer/app/components/wallet/settings/WalletSettings.js

```diff
@@ -252,8 +254,10 @@ export default class WalletSettings extends Component<Props, State> {
       onDelegateClick,
       undelegateWalletDialogContainer,
     } = this.props;
-    const isDelegating =
-      lastDelegationStakePoolStatus === WalletDelegationStatuses.DELEGATING;
+    const isDelegating = [
```

Comment:
@yakovkaravelov this is not good still.
If the current `delegationStakePoolStatus ` is DELEGATING and the `lastDelegationStakePoolStatus ` is UNDELEGATED, `isDelegating ` variable will be true and it should be false.